docs/evaluation/code.md

```diff
@@ -364,4 +364,47 @@ Keep in mind there is some variance between runs, so we recommend running evalua
 ### human-eval-infilling
 
 - Benchmark is defined in [`nemo_skills/dataset/human-eval-infilling/__init__.py`](https://github.com/NVIDIA-NeMo/Skills/blob/main/nemo_skills/dataset/human-eval-infilling/__init__.py)
-- Original benchmark source is [here](https://github.com/openai/human-eval-infilling).
\ No newline at end of file
+- Original benchmark source is [here](https://github.com/openai/human-eval-infilling).
+
+### IOI
```

Comment:
ioi24 is already listed above, do you want to move this there? Or remove that definition on top?